torq.q

```diff
@@ -612,12 +613,11 @@ if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
 // function to execute functions in .proc.initlist
 .proc.init:{
-	$[count .proc.initlist;
-		[{[a].lg.o[`init;"attemping to run initialisation: ",-3!a];
-		@[value;a;
-		{[x;a].lg.e[`init;x," error - failed to run initialisation: ",-3!a]}[;a]]}
-		each .proc.initlist;.proc.initlist:()];
-		.lg.o[`init;"no initialisation functions found"]];
+    if[0=count .proc.initlist;:.lg.o[`init;"no initialisation functions found"]];
+    {[a].lg.o[`init;"attemping to run initialisation: ",-3!a];
```

Comment:
Should avoid internal functions where possible, use `.Q.s` instead of `-3!` (or maybe just `string`?)

Reply:
Reason why I'm using `-3!` here is because I get a type error with `string` and `.Q.s`  adds an `\n`:

```q)).lg.o[`init;"attemping to run initialisation: ",string a]
'type
  [7]  /home/jpartridge/torq/deploy/torq.q:188: .lg.format:{[loglevel;proctype;proc;id;message] "|" sv (string .proc.cp[];string .z.h;string proctype;string proc;string loglevel;string id;message)}
                                                                                                    ^
q.lg)))\
q)).lg.o[`init;"attemping to run initialisation: ",-3!a]
2018.07.31D15:12:41.462410000|homer.aquaq.co.uk|discovery|discovery1|INF|init|attemping to run initialisation: "func[2]"
q)).lg.o[`init;"attemping to run initialisation: ",.Q.s a]
2018.07.31D15:12:50.649729000|homer.aquaq.co.uk|discovery|discovery1|INF|init|attemping to run initialisation: "func[2]"

```

Reply:
ah was thinking `a` was a symbol, guess not. I think maybe `.Q.s1` is the direct equivalent of `-3!` but I suppose that's getting into even more internal stuff. Suppose `-3!` is maybe fine in this case

Reply:
`.Q.s1` is the function for `-3!`.